assets/js/common/forms.js

```diff
@@ -306,8 +306,11 @@ function init() {
     updateSecondaryNav();
     showLocalSaveWarning();
 
-    // Launch this feature in non-prod envs (for now)
-    if (window.AppConfig.environment !== 'production') {
+    // Launch this feature in non-prod envs (for now) and don't add extra delays for automated tests
+    if (
+        window.AppConfig.environment !== 'production' &&
+        !window.AppConfig.isTestServer
```

Comment:
Not sure about this check, Cypress test should probably have the same experience as real users even if that's a touch slower. Does it add a massive delay?